<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesar</class>
 <widget class="QMainWindow" name="Caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caesar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_caesar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Caesar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Encrypt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Decrypt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileLocation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>429</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/endredi/Caesar/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>399</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>some_text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>399</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of text file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_location">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>429</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to text file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_increment">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>169</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Increment (min: -1000, max: 1000):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="increment">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="language">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>20</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>British</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norwegian</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_language">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Language:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="method">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Caesar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brutus</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_method">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption method:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_key">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate key</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_brutus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Brutus&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LoadFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>load file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Write2File">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>write to file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
